### ISSUE TITLE
jwt vs database session strategy

### ISSUE THREAD

**Issue by JeffreyLaCroix** [wpcodevo/nextauth-nextjs14-drizzle#1]
Is this on purpose to use `jwt` session strategy instead of `database` strategy with the Drizzle adapter? 
https://github.com/wpcodevo/nextauth-nextjs14-drizzle/blob/6d861b2e7506726378cff5406553abf2cdac2521/auth.ts#L10

**Comment by richard-aoede** [comment]
Yes I believe NextAuth in general is tricky to implement on edge runtimes at the moment with all these new ORM's.
